pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -53,11 +53,20 @@ func (np *Processor) processNewEpoch(ev netmapEvent.NewEpoch) {
 	mapChanged := !slices.EqualFunc(oldMap.Nodes(), networkMap.Nodes(), func(i1 netmap.NodeInfo, i2 netmap.NodeInfo) bool {
 		return bytes.Equal(i1.PublicKey(), i2.PublicKey())
 	})
+	if np.forceContainersListUpdate.Load() {
+		l.Info("forcing Container contract lists update...")
+
+		// fixing missing container lists from 0.45.0 release; for 0.49.0 only,
```

Comment:
What's gonna be the behavior after the update but before epoch tick? Failure to submit estimations?